retail/retail-java-applications/dashboards/README.md

```diff
@@ -0,0 +1,67 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+# Overview
+This folder contains a dashboard with several charts that will be useful for monitoring your application.
+
+You can deploy the dashboard to your Cloud Monitoring workspace by invoking the Cloud Monitoring API using ```curl``` or the ```gcloud``` command-line tool.  
+
+## Before you begin
+Ensure that you already have a Cloud Monitoring Workspace, and have the host project ID of your Workspace ready.
+
+You also need to have the ```Monitoring Dashboard Configuration Editor``` role. If you intend to use the ```gcloud``` command-line tool, you also need to have the ```Service Usage Consumer``` role.
+
+## Setup ##
+1. Create an environment variable to hold the ID of your Cloud Monitoring Workspace: 
+```
+PROJECT_ID=a-gcp-project-12345
+```
+
+2. Authenticate to Cloud SDK:
+```
+gcloud auth login
```

Comment:
I think all these need to be put into a script which calls out to gcloud. 

The goal is to show a deployment artifact, but ok to have these commands outlined in the readme to describe what the deployment artifact would do.